src/main/java/com/sayurbox/kale/abtest/command/GetAllUniverseAllocationsCommand.java

```diff
@@ -39,7 +43,10 @@ public List<GetUniverseAllocationResponse> getFallback() {
     @Override
     protected Request createRequest() {
         String url = String.format(ENDPOINT, baseUrl, userId);
-        return new Request.Builder().post(RequestBody.create(null, new byte[]{})).url(url).build();
+        byte[] body = gson.toJson(properties, propertiesType).getBytes();
+        MediaType contentType = MediaType.parse("application/json");
+
+        return new Request.Builder().post(RequestBody.create(contentType, body)).url(url).build();
```

Comment:
RequestBody bs dicreate dr string json aja hrsnya. gk perlu jd bytes. biar lebih singkat. hehehh

`RequestBody.create(contentType, gson.toJson(this.properties))`